reports/management/commands/sync_enrolment_reports.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+from reports.services import sync_enrolment_reports
+
+
+class Command(BaseCommand):
+    help = (
+        "Synchronize enrolment reports from enrolment table. "
+        + "Latest sync time will be determined from "
+        + "updated_at -field in a enrolment_reports table. "
+        + "Enrolments after that date will be updated "
+        + "if the enrolment status has changed "
+        + "and created if they are totally missing from the table content."
```

Comment:
AFAIK, since you have parentheses here you can also do this without the plus signs.
```suggestion
        "Synchronize enrolment reports from enrolment table. "
        "Latest sync time will be determined from "
        "updated_at -field in a enrolment_reports table. "
        "Enrolments after that date will be updated "
        "if the enrolment status has changed "
        "and created if they are totally missing from the table content."
```

Reply:
okay, didn't know. It sure looks better